sdks/go/pkg/beam/runners/universal/universal.go

```diff
@@ -82,6 +82,54 @@ func Execute(ctx context.Context, p *beam.Pipeline) error {
 		return errors.WithContextf(err, "generating model pipeline")
 	}
 
+	// Adding Expanded transforms to their counterparts in the Pipeline
+	for id, external := range p.ExpandedTransforms {
```

Comment:
Similarly, this code should be moved into graphx/translate.go (though TBH this is substantial enough that having things isolated in function in a graphx/xlang.go which are then called in translate.go would be a good move.)

Reply:
I've reorganized much of the code. Your opinions would be helpful to understand which of them are not relevant.